bin/nix.mjs

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env node
 'use strict'
-import '../dist/nx.mjs'
+import '../dist/nix.mjs'
```

Comment:
Thank you @joaosamouco for the proposed changes.

The permissions on this file changed. 

![image](https://user-images.githubusercontent.com/2649772/218112892-5b6bceaf-45f9-4982-814b-74c843d18de4.png)

I don't know if this was intended, so just pointing it out.

Reply:
Thanks for pointing that out. I didn't notice it!

I just checked and all other files on `bin` have `755` so I think these are the correct permissions.

Reply:
So I think you don't need it to be `755` since npm changes it to `755` on install, but this will make it easier to test